staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -78,6 +78,9 @@ type Config struct {
 	// this to confirm the proxy's identity
 	ProxyClientCert []byte
 	ProxyClientKey  []byte
+
+	// Indicates if the Aggregator should send to the cluster IP (false) or route to the endpoints IP (true)
+	EnableAggregatorRouting bool
```

Comment:
ok, considering where else we will need this logic, I think I agree that this should have an interface passed in.